new/utils/createCommonFiles.js

```diff
@@ -15,35 +14,44 @@ const createCommonFilesAndFolders = () => {
   console.log(
     "Creating the project and downloading packages, this may take a moment"
   );
-  // creates new project folder first, this is important for all new projects
-  helpers.mkdirSync(``);
-  try {
-    process.chdir(`./${store.name}`)
-    execSync('git init')
-    process.chdir('../')
-  } catch (err) {
+  try{
+    // creates new project folder first, this is important for all new projects
+    helpers.mkdirSync(``)
+    try {
+      process.chdir(`./${store.name}`)
+      execSync('git init')
+      process.chdir('../')
+    } catch (err) {
+      if (store.env === 'development') {
```

Comment:
You should do a process.chdir(`../`) because if it failed to init it won't have hit the cd .. line on 23